cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/preview/DefaultPreviewStore.java

```diff
@@ -62,113 +47,49 @@
   private static final byte[] TRACER = Bytes.toBytes("t");
   private static final byte[] PROPERTY = Bytes.toBytes("p");
   private static final byte[] VALUE = Bytes.toBytes("v");
-  private static final byte[][] COLUMNS = new byte[][] {TRACER, PROPERTY, VALUE };
 
-  private final DatasetFramework dsFramework;
-  private final Transactional transactional;
   private final Gson gson = new Gson();
   private final AtomicLong counter = new AtomicLong(0L);
 
+  private final LevelDBTableCore table;
```

Comment:
So we are getting rid of abstraction? How does it work in distributed mode down the road? Gotta rewrite?